docs/GC Rooting Guide.md

```diff
@@ -17,12 +17,12 @@ The main types of GC thing pointer are:
 - `JSObject*`
 - `JSString*`
 - `JSScript*`
-- `jsid`
+- `JS::PropertyKey (aka jsid)
 - `JSFunction*`
 - `JS::Symbol*`
 
-Note that `JS::Value` and `jsid` can contain pointers internally even
-though they are not a normal pointer type, hence their inclusion in this
+Note that `JS::Value` and `JS::PropertyKey` can contain pointers internally even
+though they are normal pointer types, hence their inclusion in this
```

Comment:
I don't think they're normal pointer types. Seems like the "not" was accidentally removed.

Reply:
Oops, thanks. Fixed.